doc/src/index.md

```diff
@@ -46,7 +58,9 @@ with performance comparable to traditional statically-typed languages.
 Because Julia's compiler is different from the interpreters used for languages like Python or
 R, you may find that Julia's performance is unintuitive at first. If you find that something is
 slow, we highly recommend reading through the [Performance Tips](@ref man-performance-tips) section before trying anything
-else. Once you understand how Julia works, it's easy to write code that's nearly as fast as C.
+else. Once you understand how Julia works, it is easy to write code that is nearly as fast as C.
+
+### [Julia compared to other languages](@id man-julia-compared-other-languages)
```

Comment:
We should be consistent with capitalization, either let's make all the words caps, or just the first word in the heading. Not sure which follows the rest of the docs better.

Reply:
I am not a fan of capitalization in headings.

Reply:
It looks like, based on my look through the docs, all H2 tags have the words all capitalized, so that is what I did here. As to when to use an H2 vs H3 (where only the first word is capitalized), now that is a mystery to me. Either way, we can just merge this and change it later if someone wants to modify it for good reason.